core/lib/mno_enterprise/concerns/controllers/auth/sessions_controller.rb

```diff
@@ -29,23 +29,41 @@ module ClassMethods
   # end
   
   # POST /resource/sign_in
-  # def create
-  #   super
-  # end
-  # def create
-  #   self.resource = warden.authenticate!(auth_options)
-  #   set_flash_message(:notice, :signed_in) if is_flashing_format?
-  #   sign_in(resource_name, resource)
-  #   yield resource if block_given?
-  #   respond_with resource, location: after_sign_in_path_for(resource)
-  # end
+  def create
```

Comment:
Backward incompatibility as this:
- no longer yield the resource to a block if provided.
- use @user instead of self.resource

Reply:
@steven-chang-maestrano why is this not addressed after 13 days?
doing it myself